src/transformers/generation/configuration_utils.py

```diff
@@ -611,7 +611,7 @@ def validate(self, is_init=False):
         if self.max_new_tokens is not None and self.max_new_tokens <= 0:
             raise ValueError(f"`max_new_tokens` must be greater than 0, but is {self.max_new_tokens}.")
         if self.pad_token_id is not None and self.pad_token_id < 0:
-            warnings.warn(
```

Comment:
These warnings, which show up in `pipelines` and `generate` are too verbose.

E.g. qwen and llama set default values for `top_p`. When we call `model.generate(..., do_sample=False)` a warning was being printed, because we shouldn't touch `top_p` with `do_sample=False`. This lowers the severity to avoid printing many messages by default.

In the future, I want to add some option to strictly validate these. But let's lower the noise for now :)

Reply:
I like that!

Reply:
(solved separately in #38179 , since the change was more complex than simply lowering the severity of the thrown warnings)